pkg/api/schemas/language.go

```diff
@@ -41,15 +42,93 @@ type ChatMessage struct {
 	Name string `json:"name,omitempty"`
 }
 
-// ChatChoice is a choice in a chat response.
-type ChatChoice struct {
+// OpenAI Chat Response
+// TODO: Should this live here?
```

Comment:
@mkrueger12 I would move it under OpenAI package. `api/schemas` was meant to be for Glide API schemas (e.g. unified schemas for whatever endpoints we provide).